akismet-api.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "akismet-api" {
+  // Type definitions for akismet-api 5.2.1
+  // Definitions by: Jason Florentino https://github.com/jasonflorentino
```

Comment:
Could we please remove these header comments? It'd better match the style in the rest of the codebase. I am happy to maintain this file but would prefer to not have the version number in here that has to be updated every time we release. Likewise, I'd be more than happy to include credits for your contribution here but a better place would be under the "Credits" section of the README than the file header IMO, since many people may end up ending this file over time. Is that alright with you?

Reply:
That sounds good to me! I was just pulling from the TS template for a declaration file 😅

Reply:
Ok great! 😄  Feel free to add that credits entry to the README as part of this PR or I can also do it in a follow up one.

Reply:
No worries Chris! No need to update the README. I'm content with seeing my avatar in the little side bar haha. But feel free to @me if these types need updating in the future. I'd be happy to help.

Reply:
Thanks a bunch!